pkg/api/controller_test.go

```diff
@@ -971,6 +971,93 @@ func TestTLSWithBasicAuthAllowReadAccess(t *testing.T) {
 	})
 }
 
+func TestTLSWithDefaultUserPermission(t *testing.T) {
```

Comment:
in this test we need to test that tls username works, not anonymous.
we need a certificate with non nil CN or subjectDN and put that into user policy.

ex *.mydomain.com
```
				AuthorizationAllRepos: config.PolicyGroup{
					Policies: []config.Policy{
						{
							Users:   []string{"*.mydomain.com"},
							Actions: []string{"read"},
						},
					}
```